AsyncDisplayKit/Layout/ASLayoutSpec.mm

```diff
@@ -156,14 +162,21 @@ - (void)setChildren:(NSArray *)children
 
 - (id<ASLayoutable>)child
 {
-  return _child;
+  return [_children firstObject];
 }
 
 - (NSArray *)children
 {
   return [_children copy];
```

Comment:
@rcancro because editing is not common, we should probably keep it immutable and remove the copy